lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/driver/VortexWorkerManager.java

```diff
@@ -35,7 +35,7 @@
 class VortexWorkerManager {
```

Comment:
Does a single thread always execute a method of `VortexWorkerManager`? 
Can multiple threads call `launchTasklet`?